pkgs/by-name/ra/redbean/package.nix

```diff
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchurl }:
+
+let
+  libPath = lib.makeLibraryPath [ stdenv.cc.cc  ];
+  arch =
+    if stdenv.is64bit
+    then "x86_64"
+    else "x86";
+in
+stdenv.mkDerivation rec {
+  pname = "Redbean";
+  version = "2.2";
+
+  src = fetchurl {
+    url = "https://redbean.dev/redbean-${version}.com";
+    sha256 = "db8fc7cc5a7703b7ccb830a366eb69e728fc7892fd3ecc093c089d837aa5b91b";
+  };
+
+  unpackPhase = "cp $src redbean.com";
+
+  nativeBuildInputs = [];
```

Comment:
You don't need this either.

Reply:
indeed, the nativeBuildInputs can be omitted.
Omitting the unpackPhase however triggers this error, so I'm leaving it in:

```
$ nix-build -E 'with import <nixpkgs> { }; callPackage ./pkgs/by-name/ra/redbean/package.nix { }'
this derivation will be built:
  /nix/store/ryvjak4scd2d6z0hylfx98p2p1vxylnl-redbean-2.2.drv
building '/nix/store/ryvjak4scd2d6z0hylfx98p2p1vxylnl-redbean-2.2.drv'...
unpacking sources
unpacking source archive /nix/store/0dv1j8vkim8l6scn96mxbc1g9hw99q6n-redbean-2.2.com
do not know how to unpack source archive /nix/store/0dv1j8vkim8l6scn96mxbc1g9hw99q6n-redbean-2.2.com
error: builder for '/nix/store/ryvjak4scd2d6z0hylfx98p2p1vxylnl-redbean-2.2.drv' failed with exit code 1;
       last 3 log lines:
       > unpacking sources
       > unpacking source archive /nix/store/0dv1j8vkim8l6scn96mxbc1g9hw99q6n-redbean-2.2.com
       > do not know how to unpack source archive /nix/store/0dv1j8vkim8l6scn96mxbc1g9hw99q6n-redbean-2.2.com
       For full logs, run 'nix log /nix/store/ryvjak4scd2d6z0hylfx98p2p1vxylnl-redbean-2.2.drv'.
```

Reply:
Hey, @coderofsalvation, you can set `dontUnpack = true;`

Reply:
done